src/dotnet-mssql/.devcontainer/mssql/postCreateCommand.sh

```diff
@@ -58,7 +58,7 @@ then
         then
```

Comment:
Can we bump the patch version for this Template? Thanks! 

https://github.com/devcontainers/templates/blob/6cc93db08951f56bb1394fda4553b98733d0cb61/src/dotnet-mssql/devcontainer-template.json#L3